deepface/extendedmodels/Gender.py

```diff
@@ -9,14 +9,17 @@
 # -------------------------------------
 # dependency configurations
 
+
 tf_version = int(tf.__version__.split(".", maxsplit=1)[0])
 
 if tf_version == 1:
     from keras.models import Model, Sequential
     from keras.layers import Convolution2D, Flatten, Activation
 elif tf_version == 2:
-    from tensorflow.keras.models import Model, Sequential
-    from tensorflow.keras.layers import Convolution2D, Flatten, Activation
+    # from tensorflow.keras.models import Model, Sequential
+    # from tensorflow.keras.layers import Convolution2D, Flatten, Activation
+    from keras.models import Model, Sequential
```

Comment:
if you delete these dependencies, then tf2 users will not be able to use deepface anymore. the imports you just added if tf1 as seen.